crates/accelerate/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "qiskit_accelerate"
+# The following options can be inherited with (e.g.) `version.workspace = true` once we hit Rust
+# 1.64.  Until then, keep in sync with the root `Cargo.toml`.
+version = "0.24.0"
+edition = "2021"
+rust-version = "1.61"
+license-file = "LICENSE.txt"
+
+[lib]
+name = "qiskit_accelerate"
+crate-type = ["cdylib"]
+
+[dependencies]
+rayon = "1.7"
+numpy = "0.18.0"
+rand = "0.8"
+rand_pcg = "0.3"
+rand_distr = "0.4.3"
+ahash = "0.8.3"
+num-complex = "0.4"
+num-bigint = "0.4"
+rustworkx-core = "0.12"
+
+# The base version of PyO3 and setting a minimum feature set (e.g. probably just 'extension-module')
+# can be done in the workspace and inherited once we hit Rust 1.64.
```

Comment:
Well this assumes we'll never have shared rust crates across our Python extension crates. A use case I could see is maybe some parser utils for OQ3 and OQ2 that we want to reuse between two separate crates (this is similar to how we use rustworkx-core in rustworkx).

Reply:
Each subpackage still has to opt in to inherit a dependency, such as by
```toml
[dependencies]
pyo3 = {"workspace" = true, "features" = ["additive", "with", "inherited", "ones"]}
```